modules/ossm-release-notes-istio-traffic-management.adoc

```diff
@@ -0,0 +1,45 @@
+////
+Module included in the following assemblies:
+* service-mesh-docs-main/ossm-release-notes-support-tables-assembly.adoc
+////
+
+//Post GA: might be worth considering putting tables into their own modules. Might easier to maintain. Warrants further discussion post GA.
+
+:_mod-docs-content-type: REFERENCE
+[id="istio-traffic-management_{context}"]
+= Istio traffic management
+
+[cols="1,1"]
+|===
+| Feature | Status
+
+| Protocols: HTTP1.1/HTTP2/HTTPS/gRPC/TCP/TLS
+| GA
+
+| Traffic Control: label/content based routing, traffic shifting
+| GA
+
+| VirtualService, DestinationRule and ServiceEntry
```

Comment:
I'm not going to go through all of these and make suggestions, since I'm not as familiar with the items. But I think that some of these (like these ones) should likely be in backticks. And some that are capitalized (like "DNS Proxying" and "Gateway Injection") should be lowercase/sentence case. Recommend checking through these and seeing which could be adjusted